pkg/shell/nav.tsx

```diff
@@ -366,6 +412,7 @@
             target: {
                 host: current_machine.address,
                 path: current_machine_manifest_items.items.apps.path,
+                hash: "/",
```

Comment:
It's not intended to be a bug fix, hmm. `hash: "/"` should be the same as omitting hash entirely, but Location requires it. I need to sort this out, in general.
Most places are guaranteed to give you a fully populated Location (such as ShellState.current_location), but functions like ShellState.jump can take a partial one. I have since learned about `Partial<Location>`, so I will go over the code to do this correctly.

Reply:
Done, this change here is no longer necessary.